spring-cloud-azure-feature-management/src/test/java/com/microsoft/azure/spring/cloud/feature/manager/feature/filters/TargetingFilterTest.java

```diff
@@ -0,0 +1,381 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See LICENSE in the project root for
+ * license information.
+ */
+package com.microsoft.azure.spring.cloud.feature.manager.feature.filters;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.Mockito.when;
+
+import java.util.ArrayList;
+import java.util.LinkedHashMap;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+
+import com.microsoft.azure.spring.cloud.feature.manager.TargetingException;
+import com.microsoft.azure.spring.cloud.feature.manager.entities.FeatureFilterEvaluationContext;
+import com.microsoft.azure.spring.cloud.feature.manager.targeting.ITargetingContextAccessor;
+import com.microsoft.azure.spring.cloud.feature.manager.targeting.TargetingContext;
+import com.microsoft.azure.spring.cloud.feature.manager.targeting.TargetingEvaluationOptions;
+
+import reactor.core.publisher.Mono;
+
+public class TargetingFilterTest {
```

Comment:
You could avoid the setup method using the class annotation `@RunWith(MockitoJUnitRunner.class)`